packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -1182,9 +1185,18 @@ export class PluginLedgerConnectorFabric
           throw new Error(`${fnTag} unknown ${message}`);
         }
       }
-      const outUtf8 = out.toString("utf-8");
+      let outResp = "";
+
+      switch (responseType) {
+        case "JSONstring":
```

Comment:
@barnapa No matching strings please. Please declare this as a constant and then also make sure to export it as part of the public API surface so that people who are using it can be sure that they are typing in the correct string because the compiler type-checks it.

Reply:
@petermetz Changed it as you requested.